.github/workflows/testing.yml

```diff
@@ -35,7 +35,7 @@ jobs:
         && mamba install --file requirements.d/development.txt)
         
     - name: Run tests
-      timeout-minutes: 35
+      timeout-minutes: 60
```

Comment:
The reason the test is timing out is not because it needs more time.
It seems like pytest is not giving control back after all tests are finished, maybe there is some unclosed thread still hanging around somewhere.
I had that on my PC once, pytest finished, I saw all results but I had to ctrl+C out to get control again. This is likely what is happening here.

Reply:
likly the finalizer in conftest did not terminate the processes